android/quest/src/main/java/org/smartregister/fhircore/quest/ui/login/LoginScreen.kt

```diff
@@ -404,7 +407,10 @@ fun ForgotPasswordDialog(
       )
     },
     text = {
-      Text(text = stringResource(R.string.call_supervisor, "012-3456-789"), fontSize = 16.sp)
+      Column(modifier = modifier.fillMaxWidth().padding(horizontal = 16.dp, vertical = 12.dp)) {
+        Text(text = stringResource(R.string.call_supervisor), fontSize = 16.sp)
+        Text(supervisorContactNumber.toString())
```

Comment:
The supervisor contact is nullable. Could you add a `null` && `isBlank()` check? Otherwise, if the contact is null, the string will be "null" which is incorrect phone format.